code/frontend/src/app/group-status-change/group-status-change.component.ts

```diff
@@ -183,6 +183,25 @@ export class GroupStatusChangeComponent implements OnInit {
 
     }
 
+    private getReason (statusCode: string): string {
```

Comment:
Pomysl dobry, ale to zdecydowanie powinno zostać usuniete, mamy to wczytane z bazy i gdzies wystawione przez toolsService lub workTypeService lub jakis dictionaryService

Reply:
Wydaje mi się że nie jest to pobierane z bazy danych to jest ustawienie opisu komentarza dla danego zlecenia 
![image](https://user-images.githubusercontent.com/36537754/102817676-81827080-43d0-11eb-94ef-c6fc994f4a11.png)

Reply:
tak, zle spojrzalem, mamy taki freetext, mozna to ew uspojnic i wrzucic w jedno miejsce.